### PR TITLE
Add mask export args to README

### PR DESCRIPTION
**Pull request by will-moore**
Add instructions from #9 and #11.

I realise that these arguments are described in the cli help, but I find them easier to find and understand when you can read them on the page.
Also includes hql for creating mask-map.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +45,62 @@ $ omero zarr export 1 --bf --output /home/user/zarr_files
...
+
+The default behaviour is to export all masks on the Image to a single 5D
+"labelled" zarr array, with a different value for each mask Shape.
+An exception will be thrown if any of the masks overlap.
+
+To handle overlapping masks, you can export as a 6D zarr array or
+split one zarr group per ROI, using ROI ID as the group name.
+
+```
+# 6D mask saved under 1.zarr/masks/0
```
Comment (manics) [inline README.md:71]: IIUC we've decided against this and it's not compliant with the spec since it's 6D, should we remove it from the plugin?

Comment (will-moore) [reply README.md:71]: Certainly 6D mask wouldn't be handled well by `ome-zarr-py` napari plugin.

Comment (will-moore) [reply README.md:71]: @joshmoore Do we want to keep `6d` option? If so, I'll mark it as "doesn't comply with ome-zarr spec".

Comment (joshmoore) [reply README.md]: I dropped it from the README. `-h` will warn about non-compliance.

**Pull request merged**